README.md

```diff
@@ -104,12 +110,15 @@ the second capture group assigns the value `14759753350` to the `epoch` variable
 ### Hook Module
 
 The video-importer is designed to work with any service that can ingest video for event segmentation and labeling. 
-The hooks-module specifies a python module with these functions used to interact with the desired services:
+The hooks-module (`--hooks_module`) specifies a python module with these functions used to interact with the desired services:
 
 1. `register_camera` - Informs the service about a new camera that has been found
 2. `post_video_content` - Sends the video data to the segmenter via a POST
 3. `set_hook_data` - Sets hook-specific data (specific to auth, camera data, etc.)
 
+An example of the structure of these functions can be found in the [`hooks_template.py`](hooks_template.py) file, or below in 
+the following 3 subsections.
+
 
 #### Camera Registration Function
```

Comment:
Since the names of the functions are expected to be consistent, let's use the function name as the headers of each section like:
#### register_camera